content/md/first-steps/discover-the-dashboard.md

```diff
@@ -3,11 +3,15 @@ id: discover-the-dashboard
 themes: first-steps
 title: Discover the **dashboard** in details
 popular: false
-related: what-is-a-project, project-widget-dashboard, imports, exports, review-products-proposals, access-rights-on-products
+related: what-is-a-project, project-widget-dashboard, what-is-a-locale, imports, exports, review-products-proposals, access-rights-on-products
 ---
 
 # Overview
-The dashboard is the default opening page, once you've logged in.
+
+Akeneo's Dashboard allows you to easily follow up your catalog enrichment progress, review the collaborative projects advancement (number of products completed, in progress or to be done) and supervise your teammates activity with the Teamwork Assistant project's widget.   
+It will also enable you to overview the number of products complete and incomplete per channel and locale, check the last operations and access the jobs reports in one click.    
```

Comment:
```suggestion
It will also enable you to view the number of products that are complete and incomplete per channel and locale, check prior operations, and access the job reports in one click.    
```